src/TConsole.cpp

```diff
@@ -650,6 +653,69 @@ void TConsole::Command_Status(const std::string&, const std::vector<std::string>
     Application::Console().WriteRaw(Status.str());
 }
 
+void TConsole::Autocomplete_Lua(const std::string& stub, std::vector<std::string>& suggestions) {
+    auto stateNames = mLuaEngine->GetLuaStateNames();
+
+    for (const auto& name : stateNames) {
+        if (name.find(stub) == 0) {
+            suggestions.push_back("lua " + name);
+        }
+    }
+}
+
+void TConsole::Autocomplete_Kick(const std::string& stub, std::vector<std::string>& suggestions) {
+    std::string stub_lower = boost::algorithm::to_lower_copy(stub);
+
+    auto LowercaseCompare = [](std::string Name1, std::string Name2) -> bool {
```

Comment:
This can likely become a function, since we use it in multiple places - for example, a `static inline` function in this file.
Either way:
```suggestion
    auto LowercaseCompare = [](const std::string& Name1, const std::string& Name2) -> bool {
```